### PR TITLE
corrected first version of fp for comment

### PR DESCRIPTION
**Pull request by bjorn-stevens**

### CODE REVIEW

**Comment by bjorn-stevens** [comment]
If there are no grave errors, I would push this as a basis for comment as I think having it on line for comment is easier for many people.

File: orcestra_book/plans/HALO-20240907a.md [hunk 1]
```diff
@@ -188,3 +171,7 @@ vertical_preview(waypoints)
 plan.show_details()
 plan.export()
 ```
+
+```{code-cell} ipython3
+
+```
```
Comment (d70-t) [inline orcestra_book/plans/HALO-20240907a.md:177]: Probably this was a mistake:

```suggestion
```

File: orcestra_book/plans/HALO-20240907a.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ crew:
 - job: HAMP
   name: Clara Bayley
 - job: Dropsondes
-  name: Helene Glöckner
+  name: "Helene Gl\xF6ckner"
 - job: Smart/VELOX
   name: TBD
 - job: SpecMACS
-  name: tbd 
+  name: tbd
```
Comment (d70-t) [inline orcestra_book/plans/HALO-20240907a.md:16]: I'd leave this unchanged as well. Generally, I'd recommend not to mix any stylistic changes with actual content changes, as this complicates the reviewing process.

File: orcestra_book/plans/HALO-20240907a.md [hunk 3]
```diff
@@ -62,63 +62,46 @@ from orcestra.flightplan import bco, sal, mindelo, point_on_track, LatLon, IntoC
 
 lon_min, lon_max, lat_min, lat_max = -65, -5, -5, 25
 
-band    = "east"
+band    = "west"
```
Comment (d70-t) [inline orcestra_book/plans/HALO-20240907a.md:65]: I still don't get the point of this `band` variable. One could just as well use the actual point (e.g. `sal` or `bco`, or maybe better `tbpb`).

If there's any use of this variable, I'd not only switch the airport, but also the satellite ROI based on it.

Comment (bjorn-stevens) [reply orcestra_book/plans/HALO-20240907a.md:65]: I inherited it... and don't have any attachment to it

File: orcestra_book/plans/HALO-20240907a.md (hunk 2 above)
Comment (bjorn-stevens) [reply orcestra_book/plans/HALO-20240907a.md:16]: good point.. but that wasn't intentional... I am not sure how that appears at times and not at other times

Comment (d70-t) [reply orcestra_book/plans/HALO-20240907a.md:16]: It's generally a good idea to add things using `git add -p` (note the `-p`) this is a very quick interactive tool. Using it, you can check all the individual changes prior to committing, and you can easily catch and omit unintended changes.

**Comment by d70-t** [comment]
I just want to mention, the comments I put are all not grave, and they shouldn't block this PR for too long. However, they are maybe helpful to get some good practice.

**Comment by bjorn-stevens** [comment]
There is a dangling branch 0907-fp-v1 that I think we can delete... I had merge problems.

**Comment by bjorn-stevens** [comment]
> I just want to mention, the comments I put are all not grave, and they shouldn't block this PR for too long. However, they are maybe helpful to get some good practice.

that is how I interpreted them :)

**Pull request merged**